galley/pkg/config/processor/metadata/metadata.gen.go

```diff
@@ -565,6 +565,7 @@ snapshots:
       - "istio/security/v1beta1/authorizationpolicies"
       - "k8s/core/v1/namespaces"
       - "k8s/core/v1/services"
+      - "k8s/core/v1/pods"      
```

Comment:
Please add a TODO to remove this.  Galley doesn't need to serve this.

Reply:
Done.